public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
This allows robots to complete access, which is what we want, but, is there any benefits of having this file versus not having it?

Reply:
robots.txt allows you to prevent web crawlers from accessing certain URLs from the site so that the server doesn't get overloaded. It is not necessary for this PR (to make the app a PWA), I just  added it is because I felt it could be useful as the application grows. Let me know in case you want me to remove it. :)

Reply:
Let us keep it :)